conda/_vendor/auxlib/factory.py

```diff
@@ -3,6 +3,7 @@
 """
 from __future__ import absolute_import, division, print_function
 
+from conda.exceptions import CondaVerificationError
```

Comment:
Actually, undo the changes in `_vendor`.  Those shouldn't be importing from `conda` at all.